### ISSUE TITLE
[doc] Make the link to documentation shorter and memorizable?

### ISSUE THREAD

**Issue by archibate** [taichi-dev/taichi.graphics#14]
Shall we move:
https://taichi.graphics/versioned_docs/develop/documentation/advanced/meta.html
to:
https://taichi.graphics/docs/develop/advanced/meta.html

**Comment by rexwangcc** [comment]
I think that's a good idea. Actually I was hisitated about using `versioned_docs` initially since it's too long. The only concern is that if:
```diff
- https://taichi.graphics/versioned_docs/develop/documentation/advanced/meta.html
+ https://taichi.graphics/docs/develop/advanced/meta.html
```
will make the doc writers a bit more confusing about where to go to write docs, since the URL is bond to the file structure, which means all stuff under `website/docs/versioned_docs` will be moved to `website/docs/docs`.

**Comment by archibate** [comment]
So why not just rename website/docs to website?

**Comment by rexwangcc** [comment]
> So why not just move /website/docs/src to /src?

Sounds good, we could look into it, especially before more docs come in.

We didn't go with a more flat structure simply because the vuepress scaffold made some decisions around the layout, see ["VuePress follows the principle of "Convention is better than configuration". "](https://vuepress.vuejs.org/guide/directory-structure.html#directory-structure)

**Comment by rexwangcc** [comment]
Resolved by #15
